pkg/agent/route/route_linux.go

```diff
@@ -115,19 +116,46 @@ func (c *Client) Initialize(nodeConfig *config.NodeConfig, done func()) error {
 func (c *Client) initIPTablesOnce(done func()) {
 	defer done()
 	backoffTime := 2 * time.Second
+	stopCh := signals.RegisterSignalHandlers()
+	c.syncIPTablesWithRetry(backoffTime)
+	klog.Info("Successfully synced iptables")
+	klog.Info("Starting sync loop for IPTables")
+	go c.ipTablesSyncLoop(stopCh)
```

Comment:
We try to start all long-lived goroutines from https://github.com/vmware-tanzu/antrea/blob/master/cmd/antrea-agent/agent.go, please try to follow the same pattern.

Reply:
Updated the PR.